src/display/canvas.js

```diff
@@ -2275,6 +2275,21 @@ class CanvasGraphics {
 
     ctx.lineWidth = lineWidth;
 
+    if (glyphsLength && glyphs[0].isInInvalidPDFjsFont) {
```

Comment:
Given that the property in question is a `Font`-instance one, rather than one that depends e.g. the `charCode`, this shouldn't be added to each and every `Glyph`-instance like this.
Hence, please just export `isInvalidPDFjsFont` the same way that we do for all other font-properties (see e.g. `exportData` in `src/core/fonts.js`) and then do:
```suggestion
    if (font.isInvalidPDFjsFont) {
```

Note that I'm also suggesting an ever so slightly shorter name for the new property.